standalone/src/models/product-selection/product-of-selection/fields-config.ts

```diff
@@ -0,0 +1,18 @@
+import { type TModelFieldsConfig } from '@/core';
+import { ReferenceGraphql } from '@/models/commons';
+import type { TProductOfSelectionGraphql } from '../types';
+
+const commonFieldsConfig = {
+  productRef: ReferenceGraphql.presets.productReference().buildGraphql(),
+  product: null,
+  variantSelection: null,
+  variantExclusion: null,
+};
```

Comment:
No need to use a `common` constant as these props are only used once.
Please move them to the config object below.